napari/_vendor/darkdetect/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2019, Alberto Sottile
```

Comment:
Do you think it would be ok to  add a tool / script that vendors dark detect so that we can easily: 

1) make it easy to update. 
2) check that no-one directly patch it in CI.

Reply:
I dont follow @Carreau 🤔 

Could you elaborate :) ?

Reply:
You manually coped files from I believe https://pypi.org/project/darkdetect/ to this directory right ? 
What if there is a new version published ? Do we have to copy those files again ? 

Maybe something that does:

```
git clone https://github.com/albertosottile/darkdetect
cd  darkdetect
git checkout  v0.5.1
cd - 
rm -rf napari/_vendored/darkdetect
cp -r darkdetect/darkdetect/ napari/_vendored/darkdetect
cp  darkdetect/License napari/_vendored/darkdetect
```
That allow us to easily update if needs be, and in CI we can add on top of that 

```
git diff napari/_vendored/darkdetect
```

And fail if the status code is non-zero as that means someone modified the file in `napari/_vendored/darkdetect` which they should not. 

Does it make sens ?

See https://github.com/napari/napari/issues/3267 as well.

Reply:
It does, I actually wanted to know if we wanted to vendor it or just add a new dependency :) 

@napari/core-devs ?

Reply:
And yes I could add a check on CI as you suggest that is a great idea :)

Reply:
Added a new check @Carreau feel free to comment :)